github/checks.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+)
+
+// ChecksService provides access to the the Checks API in the
+// GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/checks/
+type ChecksService service
+
+// CheckRun represents a GitHub check run on a repository associated with a GitHub app
+type CheckRun struct {
+	ID           *int64          `json:"id,omitempty"`
+	HeadSHA      *string         `json:"head_sha,omitempty"`
+	ExternalID   *int64          `json:"external_id,omitempty"`
+	URL          *string         `json:"url,omitempty"`
+	HTMLURL      *string         `json:"html_url,omitempty"`
+	Status       *string         `json:"status,omitempty"`
+	Conclusion   *string         `json:"conclusion,omitempty"`
+	StartedAt    *time.Time      `json:"started_at,omitempty"`
+	CompletedAt  *time.Time      `json:"completed_at,omitempty"`
```

Comment:
I believe we want to use our `Timestamp` type instead of `time.Time` for consistency. Look at other structs, e.g.:

https://github.com/google/go-github/blob/8ea2e2657df890db8fb434a9274799d641bd698c/github/repos.go#L31-L33

(I noticed we do use `time.Time` sometimes, but that looks like an unintentional issue to me.)

Reply:
👍 . Thanks I'll make that change.